### ISSUE TITLE
Investigate error and corner case scenarios and create prototypes

### ISSUE THREAD

**Issue by aliramoss** [bcgov/nr-forests-access-management#959]
**Describe the task**
Investigate and map witch interactions could return an error message or error page.

**Acceptance Criteria**
- [ ] Map whole application finding Error scenarios and corner cases.
- [ ] Check with FE about scenarios
- [ ] Create prototypes for maped scenarios.

**Comment by gormless87** [comment]
@aliramoss - can you provide and update on this issue? Have any of the AC been met? Perhaps we can meet tomorrow or next week to discuss how the work can be broken out into separate tasks so they can be completed within a single sprint.

**Comment by aliramoss** [comment]
Link to error scenarios in Figma: 
https://www.figma.com/file/5BnF5T2ncoZVpSbCbj30Mr/FAM-mockups?type=design&node-id=3029%3A64769&mode=design&t=WNYkaMqEZkNrsVOI-1

**Comment by OlgaLiber2** [comment]
@aliramoss are we done with this ticket? can we close this?

**Comment by aliramoss** [comment]
@OlgaLiber2 Yes, we can. If any other error scenarios or corner case appear, I'll create a new ticket, but I'll probably create prototypes in same Figma page.
